classes/userGroup/UserGroup.php

```diff
@@ -242,15 +244,10 @@ public function getPermitMetadataEdit()
 
     /**
      * Setter for permitMetadataEdit attribute.
-     *
-     * @param bool $permitMetadataEdit
      */
-    public function setPermitMetadataEdit($permitMetadataEdit)
+    public function setPermitMetadataEdit(?bool $permitMetadataEdit)
```

Comment:
Same as above -- is the `null` state necessary?

Reply:
Changed